Changelog.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.5.1 (Dec 2022)
+* Now support Python3.10
+* Fix bug when channels of not set then the channels is decided by the default device instead of the actual device
```

Comment:
> Fix bug when channels of not set then the channels

Bugfix: When number of channels had not been set it was determined by the default device instead of the actual one